### ISSUE TITLE
TypeScript SDK approach

### ISSUE THREAD

**Issue by whitep4nth3r** [sailhouse/docs#8]
The docs mention the TypeScript SDK, yet provide an example to connect to Sailhouse in JavaScript.

<img width="826" alt="image" src="https://github.com/sailhouse/docs/assets/52798353/dd16d6bd-abb2-4e4e-8c4f-fa7b54b476bf">

Should we name this the JavaScript SDK with TypeScript support?

If it has full TypeScript support, do we have a types package included or separate? 

Just mentioning TypeScript might scare newer developers away. The fact that I used the TypeScript SDK just by writing JavaScript is a bonus and it would be a benefit to mention/reword as such.

**Comment by estephinson** [comment]
Yeah this is a great point - it's a TS friendly library, but as always, it's just our lil JavaScript friend under the hood.

I've updated the docs to be a bit more inclusive of JS in the titles, and referenced it directly on the homepage. I'm still keen to use the word TypeScript from the out to highlight it is typed out of the box, but reassuring people that you can fallback to our trusty untyped friend if you would like!
